.travis.yml

```diff
@@ -14,5 +14,8 @@ cache:
   directories:
     - "~/.npm"
 
-before_script:
-  - npx danger ci
+jobs:
+  include:
+    - stage: danger
+      node_js: "10"
```

Comment:
<!--
  6 failure:  statements are no..., tab indentation e..., unexpected label ..., unused expression..., unused expression..., unused expression...
  3 warning:  Missing semicolon..., Missing semicolon..., Missing semicolon...
  
  
  DangerID: danger-id-Danger;
  File: .travis.yml;
  Line: 20;
-->
- :no_entry_sign: statements are not aligned line - .travis.yml (align)
- :no_entry_sign: tab indentation expected line - .travis.yml (indent)
- :no_entry_sign: unexpected label on statement line - .travis.yml (label-position)
- :no_entry_sign: unused expression, expected an assignment or function call line - .travis.yml (no-unused-expression)
- :no_entry_sign: unused expression, expected an assignment or function call line - .travis.yml (no-unused-expression)
- :no_entry_sign: unused expression, expected an assignment or function call line - .travis.yml (no-unused-expression)
- :warning: Missing semicolon line - .travis.yml (semicolon)
- :warning: Missing semicolon line - .travis.yml (semicolon)
- :warning: Missing semicolon line - .travis.yml (semicolon)